kolibri/core/assets/src/utils/errorReportUtils.js

```diff
@@ -6,13 +8,36 @@ class ErrorReport {
   getErrorReport() {
     throw new Error('getErrorReport() method must be implemented.');
   }
+
+  getContext() {
+    return {
+      browser: browser,
+      os: os,
+      device: {
+        ...device,
+        is_touch_device: isTouchDevice,
+        screen: {
+          width: window.innerWidth,
```

Comment:
This stackoverflow answer suggests a different way of getting this info:

https://stackoverflow.com/a/2242100

I think it would be good to double check what we gather here, land would be interesting to maybe gather both the overall screen resolution as well as the available resolution to the page?